docs/theory.rst

```diff
@@ -200,9 +200,10 @@ The usual manifestation of these roundoff errors is high-amplitude small-scale n
     In such cases, the user has a few options to try to regain stability.
 
     1. If the data being filtered is single-precision, it might help to promote it to double precision (or higher) before filtering.
-    2. The user can also try reducing ``n_steps``, but must not reduce it too much or the resulting filter will not behave as expected.
-    3. Users might elect to *coarsen* their data before filtering, i.e. to reduce the resolution of the input data before applying the filter. This has the effect of increasing the grid size, and thus decreasing the gap between the filter scale and the grid scale.
-    4. The final option is simply to use a different approach to filtering, not based on ``gcm-filters``.
+    2. Applying a Gaussian filter with ``filter_scale`` :math:`L` twice is the same as applying a Gaussian filter with ``filter_scale`` :math:`\sqrt{2} L` once. More generally, one can break a single large-scale Gaussian filter up into several Gaussian filters each of which has a smaller filter scale. The Filter class has an argument ``n_iterations`` that will automatically split a Gaussian filter into a sequence of smaller filters, each of which is less sensitive to roundoff errors. If a user is encountering instability with the standard Gaussian filter, the user can try setting ``n_iterations`` to an integer greater than 1. The user should input their desired filter scale, and then the code will automatically adjust it to reflect the scale of the smaller constituent filters. Applying the Taper filter twice is not equivalent to applying it once with a different scale, so this method only works for the Gaussian filter.
```

Comment:
suggested syntax change: Filter class --> :py:class:`gcm_filters.Filter` class

"... that will automatically split a Gaussian filter into a sequence of `n_iterations` smaller filters, ..."

Reply:
We could be even more specific and replace the sentence

"Applying a Gaussian filter with ``filter_scale`` :math:`L` twice is the same as applying a Gaussian filter with ``filter_scale`` :math:`\sqrt{2} L` once."

by

"Applying a Gaussian filter with ``filter_scale`` :math:`L` `N` times is the same as applying a Gaussian filter with ``filter_scale`` :math:`\sqrt{N} L` once."

Reply:
I'm updating the syntax. For the second suggestion, I kind of like it the way it is. If I updated the sentence a suggested then I would need to delete the next sentence, which basically says the same thing in words. Also, the proof of the `N` step case is just a repeated application of the two-step, so starting directly from `N` sort of hides how the proof works. Not that it will matter to most readers. I can change it if you (or other reviewers) feel strongly about it.

Reply:
Independent of whether we write about the two-step or `N`-step case in the `theory.rst`, would it make sense to have the "proof" in the documentation? (I think it's a one-liner, right?) Maybe we could outsource it to the [numerical instability example notebook](https://gcm-filters.readthedocs.io/en/latest/examples/example_numerical_instability.html). I anticipate users asking about it because the splitting is not described in the Grooms et al. 2021 paper. It would also clarify why it doesn't work for the Taper.

Reply:
It would have been super easy to add to the JAMES paper, if only we'd thought of it... Adding it to the theory.rst would take some space: it's a one-line proof if all the notation is defined, but we don't have the notation defined yet. 

When I was cooking up a test to compare the iterated and original filters, I derived an error bound. (the new test just checks that the error bound is satisfied.) I could add a new "chapter" to the documentation (not a new section of theory.rst) that proves the equivalence, derives the error bound, and explains why it won't work for the Taper filter? Not sure the best way to do this.

Reply:
Maybe a new subsection of theory.rst would be best. Are you worried it will get too long?

Reply:
In the latest commit I added a new section to theory.rst. It's a bit long; let me know what you think.